scripts/lib.sh

```diff
@@ -134,18 +134,22 @@ requiredenvs() {
 }
 
 gh_auth() {
-	local fail=0
-	if [[ "${CODER:-}" == "true" ]]; then
-		if ! output=$(coder external-auth access-token github 2>&1); then
-			log "ERROR: Could not authenticate with GitHub."
-			log "$output"
-			fail=1
+	if [[ -z ${GITHUB_TOKEN:-} ]]; then
+		if [[ -n ${GH_TOKEN:-} ]]; then
```

Comment:
We should always get the new token. The token stored in `GITHUB_TOKEN` may be stale, so I opted to refresh this even though it's set.

`coder external-auth access-token github` handles the auto-refreshing automatically.